packages/ng/libraries/core/src/style/theme/_select.theme.scss

```diff
@@ -3,6 +3,21 @@ $select: (
 	chip: (
 		background: _theme("commons.background.dark"),
 		text: _color("text.default")
+	),
+	input: (
+		padding-horizontal: .5rem,
+		padding-vertical: .45rem,
```

Comment:
Dans ce cas, on devrait faire référence à la valeur de thème de textfield ici. Sinon, changer un textfield demanderait de repasser partout. Hors finalement, ce n'est qu'un textfield comme un autre.

Reply:
english

i know, what we could do is having textfield theme and select theme both reference a theme variable in `commons`

Reply:
also, a `lu-select` should only have the style of a textfield when incorporrated in a `.textfield`

```html
<div class="textfield">
  <lu-select class="textfield-input">...</lu-select>
  <label class="textfield-label">label</label>
</div>
```

as a result i dont know why having the `_component('textfield.padding.right')` [here](https://github.com/LuccaSA/lucca-front/pull/830/files#diff-bd28bc5b6bd826013ab8b2057fb439a3L17-L18) is mandatory